app/frontend/package.json

```
@@ -17,9 +17,10 @@
     "watch": "./node_modules/.bin/webpack --config build/webpack.dev-watch.conf.js --watch --display-entrypoints"
   },
   "dependencies": {
+    "@bcgov/bootstrap-theme": "git+https://github.com/bcgov/bootstrap-theme.git",
```

I wonder if it's work specifying the latest commit? They aren't using any tags in that repo, and we probably don't want to be caught unawares?
```suggestion
    "@bcgov/bootstrap-theme": "git+https://github.com/bcgov/bootstrap-theme.git#7809a4fae4f2e44686bda62df17540e487fa5413",
```